### PR TITLE
Add StartHere.jl, remove Hacktoberfest 2020

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

File: template.html [hunk 1]
```diff
@@ -14,8 +14,8 @@ <h1>Julia Hacktoberfest</h1>
 <div id="resources">
     <h2>Useful Resources</h2>
     <ul>
-        <li><a target="_blank" rel="noopener" href="https://hacktoberfest.digitalocean.com/">Hacktoberfest 2020</a> by DigitalOcean</li>
         <li><a target="_blank" rel="noopener" href="https://kshyatt.github.io/post/firstjuliapr/">Making a first Julia pull request</a> by <a target="_blank" rel="noopener" href="https://twitter.com/kslimes">Katharine Hyatt</a></li>
+        <li><a target="_blank" rel="noopener" href="https://github.com/miguelraz/StartHere.jl">StartHere.jl</a> by <a target="_blank" rel="noopener" href="https://twitter.com/miguelraz_">Miguel Raz</a></li>
```
Comment (SaschaMann) [inline template.html]: @miguelraz could you check if this info is correct? :)

Comment (miguelraz) [inline template.html:18]: Yup - although my full name/surname is "Miguel Raz Guzmán".

**Comment by miguelraz** [comment]
This LGTM. Should I just squash and merge @SaschaMann ?

File: template.html (hunk 1 above)
Comment (miguelraz) [reply template.html]: No! My full name/surname is "Miguel Raz Guzmán". Thanks for checking though :dancers:

Comment (SaschaMann) [reply template.html]:
```suggestion
        <li><a target="_blank" rel="noopener" href="https://github.com/miguelraz/StartHere.jl">StartHere.jl</a> by <a target="_blank" rel="noopener" href="https://twitter.com/miguelraz_">Miguel Raz Guzmán</a></li>
```

Comment (SaschaMann) [reply template.html]: Sorry! Fixed it

**Pull request merged**